glib/src/gstring.rs

```diff
@@ -404,6 +404,48 @@ impl<'a> ToGlibPtr<'a, *mut c_char> for GString {
     }
 }
 
+#[doc(hidden)]
+impl<'a> FromGlibContainer<*const c_char, *const i8> for GString {
+    unsafe fn from_glib_none_num(ptr: *const i8, num: usize) -> Self {
```

Comment:
To be honest, I'm not very sure if this was the right approach at all :) I'm a bit confused what would be the differences between the 3 methods in such case?

Reply:
`none` doesn't take ownership of the input pointer, the other two would (and are the same because `char` is a basic type without memory management).

Reply:
Noted, thanks!